packages/playwright/src/generators/configuration/files/playwright.config.ts.template

```diff
@@ -13,8 +16,11 @@ import { workspaceRoot } from '@nx/devkit';
  * See https://playwright.dev/docs/test-configuration.
  */
 export default defineConfig({
-  ...nxE2EPreset(__filename, { testDir: './<%= directory %>' }),
-  /* Run your local dev server before starting the tests */<% if(webServerCommand && webServerAddress) {%> 
+  ...nxE2EPreset(__filename, { testDir: './<>' }),
+  /* Run your local dev server before starting the tests */<% if(webServerCommand && webServerAddress) {%>
+  use: {
+    baseURL,
```

Comment:
by default playwright sets
```ts
    use: {
      /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
      trace: 'on-first-retry',
    },
```
unsure if we also want to set it here since it'll override the preset that sets it.

Reply:
Moved it to the playwright config to we can make `BASE_URL` more clear.